doc/whats_new/v1.0.rst

```diff
@@ -134,6 +134,7 @@ Changelog
 ...................
 
 - |Fix| :func:`config_context` is now threadsafe. :pr:`18736` by `Thomas Fan`_.
+- |Fix| :func:`_get_response` make error message clear. :pr:`20618` by `shinehide`_.
```

Comment:
We don't need an entry in what's new here. We can add the label in the PR to discard one of the CI